src/System.Reflection.Emit/tests/TypeBuilder/TypeBuilderSetParent.cs

```diff
@@ -11,22 +11,22 @@ public class TypeBuilderSetParent
     {
         public static IEnumerable<object[]> SetParent_TestData()
         {
-            yield return new object[] { TypeAttributes.NotPublic, typeof(string), typeof(string) };
             yield return new object[] { TypeAttributes.NotPublic, typeof(EmptyNonGenericClass), typeof(EmptyNonGenericClass) };
-            yield return new object[] { TypeAttributes.NotPublic, typeof(EmptyGenericClass<>), typeof(EmptyGenericClass<>) };
-            yield return new object[] { TypeAttributes.NotPublic, typeof(int?), typeof(int?) };
             yield return new object[] { TypeAttributes.NotPublic, typeof(object), typeof(object) };
             yield return new object[] { TypeAttributes.NotPublic, null, typeof(object) };
+            yield return new object[] { TypeAttributes.Abstract, typeof(EmptyGenericClass<int>), typeof(EmptyGenericClass<int>) };
             yield return new object[] { TypeAttributes.Interface | TypeAttributes.Abstract, null, null };
         }
 
         [Theory]
-        [MemberData(nameof(SetParent_TestData))]
```

Comment:
I don't understand this change.  You've removed all of the data for the theory?  It'll never run.